test_files/export.sickle.ts

```diff
@@ -0,0 +1,2 @@
+export {foo,bar} from './export_helper';
```

Comment:
Here you can see that we found "foo" and "bar" in the helper and rewrote the export accordingly.